### DISCUSSION TITLE
Crazyflie -- increase changes in flight characteristics depending on payload

### DISCUSSION THREAD

**Discussion by Marcusitron** [bitcraze/discussions#717] [Q&A]
Hi,

I did some experiments with different wheight / payload attached to a crazyflie2.1, with the aim to show quite a distinguishable system behavior depending on payload (show effects of changes in mass of aerial systems). 
I did expect some distinct changes in flight characteristics, for example the crazyflie flying like a "chubby bumblebee" at heavy loads. Unfortunately for me, your excellent controllers did a very good job in compensation by "simply" speeding up the motors and the characteristics did only change in a very minor range.    

My question is: Is there any chance to modify controller firmware in a way, that the crazyflie does not adapt so perfectly to payload changes?
If yes, where can I find this part of the code in the firmware?   

thank you very much! 
Marcus

**Comment by tobbeanton** [comment]
Have you tried tuning down the PID controller? Can be done in the [tuning tab](https://www.bitcraze.io/documentation/repository/crazyflie-clients-python/master/userguides/userguide_client/tuning_tab/).
